Source/ARTHttp.m

```diff
@@ -51,7 +52,7 @@ - (void)executeRequest:(NSMutableURLRequest *)request completion:(void (^)(NSHTT
         } else {
             [self.logger debug:@"%@ %@: statusCode %ld", request.HTTPMethod, request.URL.absoluteString, (long)httpResponse.statusCode];
             [self.logger verbose:@"Headers %@", httpResponse.allHeaderFields];
-            NSString *headerErrorMessage = httpResponse.allHeaderFields[@"X-Ably-ErrorMessage"];
+            NSString *headerErrorMessage = httpResponse.allHeaderFields[ARTHttpHeaderFieldErrorMessageKey];
```

Comment:
BTW, I hope this won't break anything. The header is definitely `X-Ably-Errormessage` but it was using `"-Ably-ErrorMessage` 🤷‍♂️